src/deepsparse/pipeline.py

```diff
@@ -541,6 +542,8 @@ def from_config(
             other runtime information that will be used across instances of the
             MultiModelEngine to provide optimal performance when running
             multiple models concurrently
+        :param logger: An optional DeepSparse Logger object for inference
```

Comment:
nit: DeepSparse Base Logger?

Reply:
I mean, some kind of logger. Every DeepSparse Logger will inherit from the Base Logger.